httpx/utils.py

```diff
@@ -107,6 +107,14 @@ def get_netrc_login(host: str) -> typing.Optional[typing.Tuple[str, str, str]]:
     return netrc_info.authenticators(host)  # type: ignore
 
 
+def get_ssl_cert_file() -> typing.Optional[str]:
+    for env_name in ("REQUESTS_CA_BUNDLE", "SSL_CERT_FILE", "CURL_CA_BUNDLE"):
```

Comment:
> We should check the three environment variables in the order above
> `(SSL_CERT_FILE, REQUESTS_CA_BUNDLE, CURL_CA_BUNDLE)`,
> as SSL_CERT_FILE is a PEP and a standard whereas the other are products of other projects.
- @sethmlarson 	https://github.com/encode/httpx/issues/306#issue-488260040

This also affects [the](https://github.com/encode/httpx/pull/307/files#diff-c73723c2d8a7c6dfecc80f229ebcef55R83) [docs](https://github.com/encode/httpx/pull/307/files#diff-c73723c2d8a7c6dfecc80f229ebcef55R88)